tools/sketch.md

```diff
@@ -1,22 +1,15 @@
 # Sketch
 Sketch is an incredibly powerful tool for creating digital interfaces. We love it because it enables us to iterate quickly and move from lo-fi to hi-fi with ease.
 
-There are ton of resources for learning the ins and outs of Sketch, here are 2:
```

Comment:
I just moved the whole "Resources" section to the bottom of that section. Seemed like a more appropriate place. But it can stay on top if you prefer.